### PR TITLE
Improved Docs

### PR DESCRIPTION
**Pull request by AnantVasu**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -69,7 +71,7 @@ To simply get counts of specific queries in a plaintext, whitespace-seperated co
 
 BUGS ETC.
 
-For bugs, remarks, suggestions etc. please email the author at j.h.stehouwer(_AT-)uvt.nl.
+For bugs, remarks, suggestions etc. please email the author at [j.h.stehouwer@uvt.nl](mailto:j.h.stehouwer@uvt.nl).
```
Comment (hermanstehouwer) [inline README]: Hi, should these days be j.h.stehouwer(_AT-)gmail.com the uvt.nl one doesn't exist anymore.
Not having the @ sign in there is to reduce spam somewhat.

The whole codebase probably needs a revamp though, as I made it 10+ years ago.

Comment (AnantVasu) [reply README]: Hi, sure! I was just trying to enhance the doc to make it more friendly. I can fork it again and update your email there if you want. 
By the way, this project was extremely helpful for learning.

**Comment by hermanstehouwer** [comment]
Hi,

I am happy you found it useful :)
I had to make it back in the day because existing libraries didn't do what
I needed for my research.
I.e. I needed to have a fast search/count in a large corpus, and all the
implementations I found were limited to DNA/RNA sequences or byte sequences.

Cheers,

Herman

**Comment by AnantVasu** [comment]
But, I got an idea about how to proceed with my project by looking into yours.
Thanks a lot for that!

**Comment by hermanstehouwer** [comment]
No worries, I am happy it was useful to someone else.

**Pull request merged**